src/Model/Table/QueuedTasksTable.php

```diff
@@ -3,7 +3,7 @@
 
 use Cake\Core\Configure;
 use Cake\Utility\Hash;
-use Tools\Model\Table\Table;
+use Cake\ORM\Table;
```

Comment:
I'm developing into an clean CakePHP installation without any special settings expecting the MySQL stuff.
>  It should be noted that this decreases performance...
and phpMyAdmin complains about it, too.